listen/property-changes.js

```diff
@@ -70,8 +68,9 @@ PropertyChanges.prototype.getOwnPropertyChangeDescriptor = function (key) {
         });
     }
     var objectPropertyChangeDescriptors = this.__propertyChangeListeners__;
-    if (!object_owns.call(objectPropertyChangeDescriptors, key)) {
+    if (!objectPropertyChangeDescriptors.hasOwnProperty(key)) {
```

Comment:
Is there definitive evidence that this sacrifice in correctness (consider key === 'hasOwnProperty') gives you a performance win? Asking for a friend.